eng/common/scripts/SemVer.ps1

```diff
@@ -18,8 +18,9 @@ class AzureEngSemanticVersion {
     [int] $PrereleaseNumber
     [bool] $IsPrerelease
     [string] $RawVersion
+    # Regex inspired but simplifie from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
```

Comment:
```suggestion
    # Regex inspired but simplified from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
```

Reply:
Lets try not to fix this with this PR. We should fix it in the azure-sdk-tools repo.

Reply:
Fixed with https://github.com/Azure/azure-sdk-tools/commit/7464a28da8ae28ed7fe7db382db3da2506b31ed4. The automation should update this PR.